### PR TITLE
Various fixes for the 'qmk' cli

### PR DESCRIPTION
**Pull request by kiilerix**
See the individual commit messages.

### CODE REVIEW

**Comment by fauxpark** [comment]
Wider key sizes will make them unreadable on default terminal widths.

**Comment by kiilerix** [comment]
> Wider key sizes will make most layouts unreadable on default terminal widths.

Yes, it will often make it wider than 80 characters. I don't know if 80 still can be called the default terminal width these days. If users should face a problem with that, it is quite obvious how to solve it.

But even then, that seems like a smaller problem than the completely unusable output as shown in [fe59798](https://github.com/qmk/qmk_firmware/pull/15846/commits/fe597987873e9c676a6323296f0aaa35c9782402) . 

Is 80 characters a hard limit for QMK?

If so, how do you look at the problem with short but unusable output? Any idea for making a compromise?

File: lib/python/qmk/info.py [hunk 1]
```diff
@@ -128,17 +128,13 @@ def _extract_features(info_data, rules):
             key = '_'.join(key.split('_')[:-1]).lower()
             value = True if value.lower() in true_values else False if value.lower() in false_values else value
 
-            if 'config_h_features' not in info_data:
```
Comment (zvecr) [inline lib/python/qmk/info.py:131]: This is actually publicly available at <https://keyboards.qmk.fm/v1/keyboards/0_sixty/info.json>. Assumptions on its use only within the codebase is not enough to ensure it should be removed.

Comment (kiilerix) [reply lib/python/qmk/info.py:131]: On the other side: Public availability proves nothing when it comes without any documentation or usage hints in repository content or history. Any use of this seems to be based on random assumptions, and it seems to be completely fair game to change without notice.

Comment (zvecr) [reply lib/python/qmk/info.py:131]: Thats not the contract we follow. Its inclusion is almost certainly not going to,
* be merged directly to master
* merged as part of a collection of fixes

Comment (kiilerix) [reply lib/python/qmk/info.py]: Ok. I dropped that changeset.

But AFAICS, the code I was trying to remove seems like tech debt. Someone who knows the intent should clarify the purpose. Without clarification, the semantics are unknown, and nobody can ever touch that code without breaking it. Especially the two first lines seems 100% pointless and thus impossible to maintain.

**Pull request closed without merging**